apps/opik-backend/src/main/java/com/comet/opik/domain/FeedbackScoreDAO.java

```diff
@@ -433,6 +452,34 @@ public Mono<List<String>> getExperimentsFeedbackScoreNames(List<UUID> experiment
         });
     }
 
+    @Override
+    @WithSpan
+    public Mono<List<String>> getProjectsFeedbackScoreNames(List<UUID> projectIds) {
+        return asyncTemplate.nonTransaction(connection -> {
+
+            ST template = new ST(SELECT_PROJECTS_FEEDBACK_SCORE_NAMES);
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                template.add("project_ids", projectIds);
+            }
+
+            var statement = connection.createStatement(template.render());
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                template.add("project_ids", projectIds);
+            }
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                statement.bind("project_ids", projectIds.toArray(UUID[]::new));
+            }
+
+            return makeMonoContextAware(bindWorkspaceIdToMono(statement))
+                    .flatMapMany(result -> result.map((row, rowMetadata) -> row.get("name", String.class)))
+                    .distinct()
+                    .collect(Collectors.toList());
```

Comment:
If you really want to show uniqueness a set is better. But probably order is nice for a returned list. In that case, you should probably sort the query results, but be mindful of performance.